src/ducks/Signals/utils/utils.js

```diff
@@ -566,3 +594,12 @@ export const validateTriggerForm = values => {
 
   return errors
 }
+
+export const couldShowChart = metric => {
+  const possibleForChart = [
```

Comment:
Why the `possibleForChart` should be initialised on every function call? Could it be moved outside the function?